src/modm/platform/adc/stm32f0/adc.hpp.in

```diff
@@ -311,7 +312,7 @@ public:
 	getInterruptFlags();
 
 	static inline void
-	acknowledgeInterruptFlag(InterruptFlag_t flags);
+	acknowledgeInterruptFlags(InterruptFlag_t flags);
```

Comment:
We should add a `[[deprecated("msg")]]` forwarding function, otherwise this will break too annoyingly.